tests/gorepo_test.go

```diff
@@ -5,11 +5,19 @@ package tests_test
 import (
 	"os"
 	"os/exec"
+	"strconv"
 	"testing"
 )
 
 // Go repository basic compiler tests, and regression tests for fixed compiler bugs.
 func TestGoRepositoryCompilerTests(t *testing.T) {
+	if b, _ := strconv.ParseBool(os.Getenv("SOURCE_MAP_SUPPORT")); os.Getenv("SOURCE_MAP_SUPPORT") != "" && !b {
+		t.Fatal("Source maps disabled, but required for this test. Use SOURCE_MAP_SUPPORT=true or unset it completely.")
```

Comment:
I considered skip, but was worried it might be bad to simply skip such an important test so easily.

Hmm...

Maybe skip if short flag is given?